### DISCUSSION TITLE
Can anyone explain why downloads are limited to 25kb/s?

### DISCUSSION THREAD

**Discussion by meikiuwu** [AndroidIDEOfficial/AndroidIDE#1631] [Q&A]
i kinda dont feel like waiting 2 hours to install an 86mb app
my download speed is usually at 300-450mb/s

**Comment by CyberXCoder-6649** [comment]
Maybe servers are slow

**Comment by binx6** [comment]
U can try using a dedicated downloader, such as **1DM+**, or using a mirror of GitHub, such as **githubfast**, which will be quick.
